### PR TITLE
add a definition for Researchers

### PR DESCRIPTION
**Pull request by CaptainSifff**
I added a quick definition of researchers. Feel free to comment and expand!

### CODE REVIEW

**Comment by mhagdorn** [comment]
I am a little surprised to see librarians mentioned in the same sentence as researchers. Maybe turning the definition around from people who rarely perform RSE activities to people who use RSE services makes a little more sense. In ITIL speak this would be the customers (those who pay the bills) and the users.  So maybe

We refer to researchers as people who are using the services provided by Research Software Engineers. These not only include academics but also people from related fields such as librarians and technicians.

**Comment by MakisH** [comment]
Another point: Wouldn't the following order make more sense?

1. Software
2. Researcher
3. Research Software
4. Research Software Engineers

**Comment by CaptainSifff** [comment]
I included the text of @mhagdorn and put it up for discussion.

**Comment by CaptainSifff** [comment]
@MakisH I'd say you would be right usually, but the current text defines "researchers" as people who are not RSEs, so I need their definition before, no?

File: competencies.md [hunk 1]
```diff
@@ -214,6 +214,11 @@ However, we also recognise that many RSEs have chosen specifically to focus on a
 role as an alternative to a traditional research role because they enjoy and wish to focus
 on the development of research software.
 
+**Researchers**: 
+We use researchers to identify people who are using the services provided by Research Software Engineers.
```
Comment (MakisH) [inline competencies.md:218]:
```suggestion
People who are using the services provided by Research Software Engineers.
```

following the same pattern

**Comment by MakisH** [comment]
> @MakisH I'd say you would be right usually, but the current text defines "researchers" as people who are not RSEs, so I need their definition before, no?

True. Then the order is fine.

File: competencies.md (hunk 1 above)
Comment (CaptainSifff) [reply competencies.md:218]: included in main.

**Pull request merged**